setup.py

```diff
@@ -18,7 +18,7 @@
 
 setup(
     name="confidant-client",
-    version="2.2.5",
+    version="2.2.6",
```

Comment:
Since you added a new set of functions, I imagine this is a minor version bump and not a patch version

```suggestion
    version="2.3.0",
```